core/src/main/scala/kafka/server/FetchSession.scala

```diff
@@ -480,7 +494,7 @@ class IncrementalFetchContext(private val time: Time,
           partitionIter.next()
         }
         debug(s"Incremental fetch context with session id ${session.id} returning " +
-          s"${partitionsToLogString(updates.keySet)}")
+            s"${partitionsToLogString(updates.keySet)}")
```

Comment:
nit: remove the whitespace and get rid of this change.

Reply:
Will do.